conduct/faq.md

```diff
@@ -0,0 +1,77 @@
+# Project Jupyter Code of Conduct - Frequently Asked Questions
+
+This FAQ attempts to address common questions and concerns around the Jupyter
+community's Code of Conduct. If you still have questions after reading it,
+please feel free to contact us at
+[*conduct@jupyter.org*](mailto:conduct@jupyter.org).
+
+### Why have you adopted a Code of Conduct?
+
+We think the Jupyter community is awesome. If you're familiar with the Jupyter
+community, you'll probably notice that the Code basically matches what we
+already do. Think of this as documentation: we're taking implicit expectations
+about behavior and making them explicit.
+
+We're doing this because the Jupyter community is growing faster than any of us
+could have anticipated. This is on balance a very positive thing, but as we've
+grown past the point where it's possible to know the whole community, we think
+it's very important to be clear about our values.
+
+We know that the Jupyter community is open, friendly, and welcoming. We want to
+make sure everyone else knows it too.
+
+### What does it mean to "adopt" a Code of Conduct?
+
+For the most part, we don't think it means large changes. We think that the text
+does a really good job describing the way the Jupyter community already conducts
+itself. We expect that most people will simply continue to behave as they have
+in the past.
+
+However, we do expect that people will abide by the spirit and words of the Code
+of Conduct when in "official" Jupyter spaces. This code has been adopted by both
+the Jupyter Steering Council and JupyterDay/JupyterCon events. That means that
+it will apply both in community spaces and at our events.
+
+In practice, this means mailing lists, the various Jupyter channels, and Jupyter
+events.
+
+### What happens if someone violates the Code of Conduct?
+
+We are all stewards of our community, and are encouraged to participate in ways
+that defend the values highlighted in this document and help others understand
+when their actions go against these values (by engaging them and directing them
+to this document if necessary). If that doesn't work, or if you need more help,
+you can contact conduct@jupyter.org. For more details please see our [Reporting
+Guidelines](*reporting_online.md*).
+
+### What about events funded by Project Jupyter?
+
+This Code of Conduct will also cover any events that Project Jupyter funds.
+
+The project requires that any events Project Jupyter funds have policies and
+procedures in place for handling harassment. It's especially important to us
+that real-life events take steps to protect the physical and mental security of
+their participants.
+
+Event organizers will enforce the Code of Conduct.
```

Comment:
We can't say that.  Enforcement [may be illegal](https://www.globalregulatoryenforcementlawblog.com/2013/04/articles/government-investigations/france-code-of-conduct-compliance-breach-is-not-automatically-a-sufficient-reason-for-employee-termination-employers-should-be-cautious-of-a-right-local-implementation-of-compliance-guidelines/). Asking a conference organiser in their country that they *have to* adopt a CoC might be asking them place themselves in an illegal situation.

The "Code of conduct" for a conference may have to be: 
  - This event is covered by local laws. Local laws do cover all the point that would be in the CoC. Here is a quick _Reminder_. [insert reminder that look like CoC]

For example, the term "Race" (french, not english spelling is the same) is [forbidden to be written in law](http://rendezvous.blogs.nytimes.com/2013/05/17/france-fights-racism-by-outlawing-race/). This CoC contains "but is not limited to, members of any **race**, ethnicity, culture...", so _this_ code of conduct would look extremely strange at a French event.

Suggestion:

> This Code of Conduct will also cover any events that Project Jupyter funds _when local legislation allows it_.

> _When allowed_, the project requires that any events Project Jupyter funds ...

> _When covered by a CoC_ , event organizers will enforce the Code of Conduct